Corona-Warn-App/src/main/res/values/strings.xml

```diff
@@ -980,6 +980,47 @@
     <!-- XACT: submission finished - illustration description, explanation image -->
     <string name="submission_done_illustration_description">"Everyone in the group is cheering because someone has shared the test result."</string>
 
+    <!-- Submission Symptoms -->
+    <!-- XHED: Page title for symptom screens -->
+    <string name="submission_symptom_title">""</string>
+    <!-- YTXT: headline text for initial symptom screen -->
+    <string name="submission_symptom_initial_headline">""</string>
+    <!-- YTXT: Bullet points for symptoms -->
+    <string-array name="submission_symptom_symptom_bullets">
+        <item>""</item>
```

Comment:
Is it normal that these are empty?

Reply:
This is a part of the translation process. Provide the german string values, and leave the default string values empty.